### PR TITLE
role to install packages

### PR DESCRIPTION
**Pull request by tareqalayan**
required for engine-deployment role

### CODE REVIEW

File: roles/ovirt-engine-install-packages/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+galaxy_info:
+  author: Tareq Alayan
+  description: Role to install packages required for deployin oVirt engine.
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/meta/main.yml]: deploying

File: roles/ovirt-engine-install-packages/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  license: Apache License 2.0
+
+  min_ansible_version: 2.3
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/meta/main.yml:15]: Let's not explicitly set fedora versions for all roles, so we don't need to maintain it

File: roles/ovirt-engine-install-packages/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+# default vars for ovirt-ovirt-engine-install-packages-packages role
+ovirt_engine_dwh: True
+ovirt_engine_version: 4.1
+ovirt-engine-extension-aaa-ldap: True
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/defaults/main.yml]: What about aaa-misc, logger etc? I just wonder whether it wouldn't be better to set a list of additional packages

File: roles/ovirt-engine-install-packages/meta/main.yml (hunk 1 above)
Comment (tareqalayan) [reply roles/ovirt-engine-install-packages/meta/main.yml]: done

File: roles/ovirt-engine-install-packages/meta/main.yml (hunk 2 above)
Comment (tareqalayan) [reply roles/ovirt-engine-install-packages/meta/main.yml:15]: ok

File: roles/ovirt-engine-install-packages/defaults/main.yml (hunk 3 above)
Comment (tareqalayan) [reply roles/ovirt-engine-install-packages/defaults/main.yml]: changing to list.

File: roles/ovirt-engine-install-packages/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Requirements
+------------
+
+Preinstalled clean environment with configured repositories.
+
+Role Variables
+--------------
+
+None
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/README.md]: You miss the documentation of variables here.

File: roles/ovirt-engine-install-packages/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+# default vars for ovirt-ovirt-engine-install-packages
+ovirt_engine_version: 4.1
+ovirt_engine_type: ovirt-engine
+packages:
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/defaults/main.yml]: please use `ovirt_engine_` prefix

File: roles/ovirt-engine-install-packages/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    state: installed
+    update_cache: yes
+  tags:
+    - skip_yum_install_ovirt_engine
+
+- name: yum install packages
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ packages | default([]) }}"
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/tasks/main.yml]: the `| default([])` is not needed as you have it set to `ovirt-engine-extensions-aaa-ldap`

File: roles/ovirt-engine-install-packages/tasks/main.yml [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+# tasks file for ovirt-engine-install-packages
+- name: yum install engine
+  yum:
+    name: "{{ ovirt_engine_type }}"
```
Comment (machacekondra) [inline roles/ovirt-engine-install-packages/tasks/main.yml]: I think the name is ovirt-engine also for downstream , no?

**Comment by KKoukiou** [comment]
@machacekondra @mwperina wdyt?

**Comment by machacekondra** [comment]
I've already mentioned somewhere that this should not be a role, but this should be part of setup role.

**Comment by KKoukiou** [comment]
@machacekondra I agree actually. I probably missed this comment. So, closing this PR.

**Comment by KKoukiou** [comment]
I don't have permission. It's Tareqs. @tareqalayan please close this PR, see reason in comments above.

**Pull request closed without merging**